turbo.json

```diff
@@ -1,5 +1,15 @@
 {
-	"globalEnv": ["CI", "COVERAGE_ENABLED", "BUILD_WITH_COVERAGE"],
+	"globalEnv": [
+		"CI",
+		"COVERAGE_ENABLED",
+		"BUILD_WITH_COVERAGE",
+		"CODECOV_TOKEN",
+		"GITHUB_ACTIONS",
+		"GITHUB_REPOSITORY",
+		"GITHUB_SHA",
+		"GITHUB_HEAD_REF",
+		"GITHUB_REF"
```

Comment:
Should we be using [passthrough envs](https://turborepo.com/docs/crafting-your-repository/using-environment-variables#passthrough-variables) for these? Else how do we make sure that e.g. every SHA doesn't bust the cache?